frontend/src/plugin/registry.tsx

```diff
@@ -101,4 +102,26 @@ export default class Registry {
   registerAppBarAction(actionName: string, actionFunc: (...args: any[]) => JSX.Element | null) {
     store.dispatch(setAppBarAction(actionName, actionFunc));
   }
+
+  /**
+   * Append the specified title and component to the details view.
+   * @param sectionName a unique name for it
+   * @param sectionFunc - a function that returns your detail view component with props
+   *                      passed into it and the section title
+   *
+   * @example
+   *
+   * ```JSX
+   * register.registerDetailsViewSection("Block I/O Latency", () => { title: 'Block I/O Latency', component: (props) => <BioLatency {...props}/>});
```

Comment:
The section name could be examplified as `biolatency`. I am not sure what we'll do with it yet, but maybe it's good to have it as an identifier.